<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>160</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="settingsButton">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ratingButton">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rating</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="changePasswordButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>ChangePassword</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>501</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BEST GAME:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMistakes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mistakes:</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorsLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
